lmcache/v1/cache_engine.py

```diff
@@ -178,6 +192,9 @@ def store(
         :raises: ValueError if the number of Falses in the mask is not a
             multiple of the chunk size.
         """
+        if self.save_only_first_rank and not self.metadata.is_first_rank():
```

Comment:
Can we add a helper function in CacheEngine with a name like `is_passive`:
```
    def is_passive(self):
        """A 'passive' CacheEngine means that the node itself will not store/retrieve 
        the data directly, but from the "active" worker (i.e., rank 0 in MLA)
        """
        return self.save_only_first_rank and not self.metadata.is_first_rank()
```

Then the if condition can be more readable:

```suggestion
        if self.is_passive():
```

Reply:
done